kombu/connection.py

```diff
@@ -173,7 +173,7 @@ def __init__(self, hostname='localhost', userid=None,
                 # e.g. sqla+mysql://root:masterkey@localhost/
                 params['transport'], params['hostname'] = \
                     hostname.split('+', 1)
-                transport = self.uri_prefix = params['transport']
+                transport = self.uri_prefix = params['transport']  # unused
```

Comment:
There's also the possibility that line 187: `params['transport'] = transport` is indented a level too much?